CIP-0135/README.md

```diff
@@ -0,0 +1,362 @@
+    - Nicholas Clarke <nicholas.clarke@tweag.io>
+Implementors: N/A
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/893
+Created: 2024-06-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+While the Cardano mainnet and other networks have proven to be highly resilient, it is necessary to proactively 
+consider the possible recovery mechanisms and procedures that may be required in the unlikely 
+event of a major failure where the network is unable to recover itself. 
+
+This CIP considers three representative scenarios and addresses specific considerations relevant 
+in each case:    
+
+Scenario 1 - __Long-Lived Network Partition__    
+Scenario 2 - __Failure to Make Blocks for an Extended Period of Time__    
+Scenario 3 - __Bad Blocks Minted on Chain__    
+
+To ensure successful recovery in the event of a chain failure, it's crucial to establish effective 
+communication channels and exercise recovery procedures in advance to familiarize the community and 
+stake pool operators (SPOs) with the process.
+
+This CIP is based on an earlier IOHK technical report that is referenced below, supplemented by internal 
+documentation and discussions that have not been publicly released. It should be considered to be a living 
+document that is reviewed and revised on a regular basis.    
+
+Note that although the focus of disaster recovery is on Cardano mainnet, since this is the greatest risk
+of loss of funds, the recovery procedures are generic and apply to other Cardano
+networks, including SanchoNet, Preview, PreProd or private networks.
+Appropriate adjustments may need to be made to reflect differences in timing or other concerns.
+
+
+## Motivation: why is this CIP necessary?
+
+This CIP is needed to familiarize stakeholders with the processes and procedures that should be 
+followed in the unlikely event that the Cardano mainnet, or another Cardano network, encounters
+a situation where the built-in on-chain recovery mechanisms fail.
+
+## Specification
+
+While the exact recovery process will depend on the unique nature of the failure, there are three main scenarios we can consider.
+
+### Scenario 1: Long-Lived Network Partition
+
+Ouroboros Praos is designed to cope with real-world networking
+conditions, in which some nodes may temporarily be disconnected from
+the network.  In this case, the network will continue to make blocks,
+perhaps at some lower chain density (reflecting the temporary loss of
+stake to the network as a whole).  As nodes rejoin the network, they
+will then participate in normal block production once again. In this
+way, the network remains resilient to changes in connectivity.
+
+If many nodes become disconnected, the network could divide into two
+or more completely disconnected parts.  Each part of the network could
+then form its own chain, backed by the stake that is participating in
+its own partition.  Under normal conditions, Praos will also deal with
```

Comment:
```suggestion
its own partition. Under normal conditions, Praos will also deal with
```

Reply:
@kevinhammond @perturbing - the world has **_not_** standardised on "once space after a period" so two spaces continue to be fine for CIPs and for everything else: https://www.chicagomanualofstyle.org/qanda/data/faq/topics/OneSpaceorTwo.html

These corrections would be like marking the term "web site" as incorrect.  This was considered the _only_ correct spelling until lots of people started misspelling it "website".  Regardless of what some people think about which spelling is canonical, neither can be marked incorrect.  It is the same way for "2 spaces after a period" and will be so for many years to come.

Therefore I'm resolving all similar issues below & we'll continue discussion if necessary here.

Reply:
Thanks, I learned something. Did not know that :)